cli/ttl.go

```diff
@@ -89,6 +91,27 @@ func ttlset() *cobra.Command {
 				_, _ = fmt.Fprintf(cmd.OutOrStdout(), "warning: ttl rounded down to %s\n", truncated)
 			}
 
+			if changed, newDeadline := changedNewDeadline(workspace, truncated); changed {
+				// For the purposes of the user, "less than a minute" is essentially the same as "immediately".
+				timeRemaining := time.Until(newDeadline).Truncate(time.Minute)
+				humanRemaining := "in " + timeRemaining.String()
+				if timeRemaining <= 0 {
+					humanRemaining = "immediately"
+				}
+				_, err = cliui.Prompt(cmd, cliui.PromptOptions{
+					Text: fmt.Sprintf(
+						"Workspace %q will be stopped %s. Are you sure?",
+						workspace.Name,
+						humanRemaining,
+					),
+					Default:   "yes",
+					IsConfirm: true,
+				})
+				if errors.Is(err, cliui.Canceled) || err != nil {
+					return nil
```

Comment:
FYI this rewrite is absolutely fine, but I had something like this in mind so that the user may know that something went wrong:

```suggestion
				if err != nil {
					if errors.Is(err, cliui.Canceled) {
						return nil
					}
					return err
```